solidity/test/LiquidityProtectionAverageRate.js

```diff
@@ -75,9 +77,10 @@ describe('LiquidityProtectionTokenRate', () => {
         );
         await liquidityProtectionSettings.setMinNetworkCompensation(new BN(3));
 
+        liquidityProtectionStore = await LiquidityProtectionStore.new();
         liquidityProtection = await LiquidityProtection.new(
             liquidityProtectionSettings.address,
-            defaultSender,
+            liquidityProtectionStore.address,
```

Comment:
You should also add `await liquidityProtectionStore.transferOwnership(liquidityProtection.address);` below, to make it 100% compatible with any other flow

Reply:
Done